interop/authzen-search-demo/app/data/subjects.server.ts

```diff
@@ -0,0 +1,15 @@
+const subjects = [
```

Comment:
We want another file that is the resource file, right?

Reply:
Yup so we can show the sample data once it is defined